try:
    raise NameError("Exception handling !!!")
except NameError:
    print("An Exception ")
    raise